src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfter.java

```diff
@@ -286,4 +286,11 @@ public void parentheses() {
 
     public static void testNoWhitespaceBeforeEllipses(String ... args) {
     }
+
+    @      interface WithWhitespace {}
```

Comment:
Please add some good cases where no violation is produced.
Also add one with line breaks after instead of space.